sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>We might need to update this. It looks like the current version of sagemaker SDK in the repo is <a href="https://github.com/aws/sagemaker-python-sdk/blob/master/VERSION" target="_blank">2.18.1</a>. I anticipate this to be upgraded to 2.19.x after the merging of wave 2 features. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>fixed</p><p> </p>

Reply:
<p>Just confirmed, sagemaker SDK will be bumped up to 2.19.0 to include the wave 2 features. We should update the minimum version here to be 2.19.0.</p>

Reply:
updated